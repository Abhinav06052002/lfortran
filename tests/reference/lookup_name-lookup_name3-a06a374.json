{
    "basename": "lookup_name-lookup_name3-a06a374",
    "cmd": "lfortran --lookup-name --no-color {infile} -o {outfile}",
    "infile": "tests/lookup_name3.f90",
    "infile_hash": "7181224f94a1ed635176c4414c3a6f8a8b82712a2f3834d4f9fb74b3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "lookup_name-lookup_name3-a06a374.stdout",
    "stdout_hash": "02bb7c20abdee50648e64e6a5adf4f41c979dd046f57374672809c01",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}