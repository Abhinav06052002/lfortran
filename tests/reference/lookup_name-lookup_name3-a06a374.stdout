[{"kind":13,"location":{"range":{"start":{"character":40,"line":2},"end":{"character":41,"line":2}},"uri":"uri"},"name":"ec","filename":"tests/lookup_name3.f90"}]